pom.xml

```diff
@@ -83,14 +83,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>3.0.0-M5</version>
+                    <configuration>
+                        <suiteXmlFiles>
+                            <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                        </suiteXmlFiles>
+                        <argLine>
+                            -javaagent:${com.browserstack:browserstack-java-sdk:jar}
+                        </argLine>
+                        <systemPropertyVariables>
+                            <cucumber.publish.quiet>true</cucumber.publish.quiet>
+                        </systemPropertyVariables>
+                    </configuration>
+            </plugin>
         </plugins>
     </build>
     <profiles>
         <profile>
             <id>sample-test</id>
-            <activation>
```

Comment:
Why are we removing this ?

Reply:
Because we are adding surefire plugin at root as we have in testNG, if user try to run using testNG-plugin then everytime this profile is getting triggered.

Reply:
in case of `mvn test` too. Moreover, suppose user try to pass features from RunnerTest file in @CucumberOptions. Then this profile taking precedence.